cityofaustin/atd-moped
# Replace `moped_proj_milestones` REST connector events with DB triggers

## Associated issues
fixes https://github.com/cityofaustin/atd-data-tech/issues/15488

## Testing
**URL to test:** 

local because it includes migrations

**Steps to test:**
this shouldn't change any ui functionality!
- go to a project's timeline and add a new milestone
- confirm that the update is registered in the activity log
- edit the milestone and check the activity log
- go to the new milestone record in the db and confirm the ``created_at``, ``updated_at``, ``created_by_user_id`` and ``updated_by_user_id`` columns are correct. there should no longer be a ``date_added`` columns
- delete the milestone and confirm in the activity log
- confirm in the db that the record was set to ``is_deleted`` is ``true
- go to the dashboard, select a followed project and click the milestone progress circle to add a new milestone, edit and delete it

note: if you are using seed data you may come across a bug where the first milestone you add isn't saved. this only seems to occur when using seed data so should only affect us devs, and it is a preexisting bug that we may want to look into!


---
#### Ship list
